src/auth.js

```diff
@@ -90,20 +91,25 @@ async function verifyAuth(token) {
       data: dbUser.content.data,
     };
 
-    logger.generic(
-      4,
-      `auth.verifyAuth() Returning Authenticated User: ${authUserObject.username}`
-    );
+    logger.generic(4, `auth.verifyAuth() returning authenticated user: ${authUserObject.username}`);
+
     return {
       ok: true,
-      content: authUserObject,
+      content: authUserObject
     };
+
   } catch (err) {
-    logger.generic(3, "auth.verifyAuth() Caught an error", {
+    logger.generic(3, "auth.verifyAuth() Cuaght an Error", {
```

Comment:
Was just skimming through, was this change intentional from `Caught` to `Cuaght`?

Reply:
Great catch, it absolutely was not lol